packages/new-polymath-issuer/src/components/Checkpoints/Container.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { Presenter } from './Presenter';
```

Comment:
Lets not fix this now, but this is a note to self to have a tslint enforce (hopefully auto-fix) imports from:

- Libraries
- Absolute Imports
- Relative Imports